### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShakedZrihen**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=d1DAEV8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,5 +73,5 @@ I'm better off when I hit the bottom
 And in the free fall I will realize
 I'm better off when I hit the bottom
 
-Lady gaga the queen
+Lady gaga the queen of Paris
```
Comment (EladKohavi) [inline README.md:76]: amsterdam?

Comment (yeelali14) [inline README.md:76]: lol

**Review by EladKohavi — CHANGES_REQUESTED** [review]
ok

**Review by yeelali14 — CHANGES_REQUESTED** [review]
lol

File: README.md [hunk 2]
```diff
@@ -73,5 +73,5 @@ I'm better off when I hit the bottom
 And in the free fall I will realize
 I'm better off when I hit the bottom
```
Comment (yeelali14) [inline README.md:74]: dasfs

**Review by yeelali14 — APPROVED** [review]
l

**Pull request closed without merging**